MCP-Cursor-راهنمای-فارسی.md

```
@@ -0,0 +1,2915 @@
+        "forecast_days": days,
+        "forecast": forecast
+    }
+
+# اجرای سرور
+if __name__ == "__main__":
+    server.run()
+```
+
+### مثال 2: سرور Node.js برای مدیریت فایل
+
+#### نصب وابستگی‌ها
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)